pkgs/tools/misc/qdl/default.nix

```
@@ -4,21 +4,22 @@
 , pkg-config
 , libxml2
 , systemd
+, libusb1
 }:
 
 stdenv.mkDerivation {
   pname   = "qdl";
-  version = "unstable-2023-04-11";
+  version = "unstable-2024-07-20";
```

```suggestion
  version = "unstable-2024-06-10";
```

The date should reflect when the last change as, i.e. the date of commit https://github.com/linux-msm/qdl/commit/cbd46184d33af597664e08aff2b9181ae2f87aa6

---

remember to reword the commit title and pr title